src/plugins/es_ui_shared/static/forms/components/fields/super_select_field.tsx

```diff
@@ -47,13 +47,13 @@ export const SuperSelectField = ({ field, euiFieldProps = { options: [] }, ...re
     >
       <EuiSuperSelect
         fullWidth
+        hasNoInitialSelection
         valueOfSelected={field.value as string}
         onChange={value => {
           field.setValue(value);
         }}
         isInvalid={isInvalid}
-        data-test-subj="superSelect"
-        options
+        data-test-subj="select"
```

Comment:
I think we should keep `superSelect` since we are already using `select` for the simple field select here https://github.com/elastic/kibana/blob/master/src/plugins/es_ui_shared/static/forms/components/fields/select_field.tsx#L54

This won't break our stuff so thank you

Reply:
I would prefer to keep it as "select" as the fact that it is a _super_ select is an implementation detail (you might decide to swap your SuperSelect with a normal select, the test should still pass).  We are not trying to get a unique id but more describing the form control, and in our tesets we would probably target it with an ancestor path (e.g. `"myForm.select"`).

It can always be overridden (and should be overridden in case there are multiple (super)select in the form.